Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -147,11 +162,13 @@ class TabLocationView: UIView {
         contentView.edges(equalTo: self)
 
         NSLayoutConstraint.activate([
-            trackingProtectionButton.widthAnchor.constraint(equalToConstant: TabLocationViewUX.TPIconSize),
+            trackingProtectionButton.widthAnchor.constraint(equalToConstant: TabLocationViewUX.ButtonSize),
```

Comment:
nit: We have been trying to move to have the UX struct inside of the view class and rename to UX